<?xml version="1.0" encoding="iso-8859-1"?>
<module name="Ft.Lib.MessageSource">
  <modification-date>2005-04-18 16:46:56</modification-date>
  <abstract>Messages for Ft.Lib</abstract>
  <description>Copyright 2004 Fourthought, Inc. (USA).
Detailed license and copyright information: http://4suite.org/COPYRIGHT
Project home, documentation, distributions: http://4suite.org/</description>
  <globals>
    <global public="yes" name="URI" id="URI">
      <value>{100: 'Invalid base URI: %(base)r cannot be used to resolve reference %(ref)r', 111: 'Invalid base URI: %(base)r cannot be used to res...f)r; the base URI must be absolute, not relative.', 120: &quot;Only a 'file' URI can be converted to an OS-specific path; URI given was %r&quot;, 121: 'A URI containing a remote host name cannot be converted to a path on posix; URI given was %r', 130: 'Error retrieving resource %(loc)r: %(msg)s', 200: 'Scheme-based resolution requires a URI with a sc... URI %(base)r nor the reference %(ref)r have one.', 201: 'The URI scheme %(scheme)s is not supported by resolver %(resolver)s', 202: 'The URI ref %(uri)r cannot be made urllib-safe o...is version of Python (IDNA encoding unsupported).', 300: 'A public ID cannot be derived from URN %(urn)r because it does not conform to RFC 3151.', 1000: 'Platform %r not supported by URI function %s'}</value>
    </global>
  </globals>
</module>
